hunspell.yaml

```diff
@@ -32,6 +33,26 @@ pipeline:
 
   - uses: strip
 
+  - runs: |
+      mkdir -p ${{targets.destdir}}/usr/share/hunspell
+      cd ${{targets.destdir}}/usr/share/hunspell
+      wget -c https://downloads.sourceforge.net/project/wordlist/speller/2020.12.07/hunspell-en_US-2020.12.07.zip
+      wget -c https://downloads.sourceforge.net/project/wordlist/speller/2020.12.07/hunspell-en_CA-2020.12.07.zip
+      wget -c https://downloads.sourceforge.net/project/wordlist/speller/2020.12.07/hunspell-en_AU-2020.12.07.zip
+      wget -c https://downloads.sourceforge.net/project/wordlist/speller/2020.12.07/hunspell-en_GB-large-2020.12.07.zip
```

Comment:
I think we'd want to find a better way to fetch these, are these available via a git checkout?  We'd maybe want to split these into subpackages using a data / range feature of melange similar to https://github.com/wolfi-dev/os/blob/120fd8a3abda7a5c729907410d683481505cf2d3/glibc.yaml#L126-L131 and https://github.com/wolfi-dev/os/blob/120fd8a3abda7a5c729907410d683481505cf2d3/glibc.yaml#L383-L389

also this change on the package would require an epoch bump for new apks to include the changes

Reply:
Thanks for the feedback @rawlingsj  :)
Found a better source of dicts from libreoffice, so I'm now using git for that

Split package is now done with data/range over `_dictionaries` :+1: